src/go/rdctl/pkg/plist/plist.go

```diff
@@ -132,13 +139,23 @@ func xmlEscapeText(s string) (string, error) {
 }
 
 // JsonToPlist converts the json settings to a reg file
+// It needs two JSON representations of the incoming data: one based on the schema,
+// and the second based on a type-free JSON parse of the data.
+// The reason for the two is that the typed schema doesn't distinguish empty arrays and dicts
+// from unspecified ones.
+// And the reason why the type-free parse isn't sufficient is that it doesn't distinguish
+// hashes (like `diagnostics.mutedChecks` from subtrees).
 func JsonToPlist(settingsBodyAsJSON string) (string, error) {
 	var settingsJSON options.ServerSettingsForJSON
+	var actualSettingsJSON map[string]interface{}
```

Comment:
This would be clearer if this is called `typedJSON` and `untypedJSON`, I think. They're both _actual_ settings, unless I'm confused somewhere…

Reply:
I'm calling it `schemaInfluencedSettingsJSON`